src/Uno.UWP/UI/Composition/SkiaCompositionSurface.skia.cs

```diff
@@ -62,7 +62,7 @@ internal void LoadFromBytes(byte[] image)
 
 		internal unsafe void SetPixels(int pixelWidth, int pixelHeight, byte[] data)
 		{
-			var info = new SKImageInfo(pixelWidth, pixelHeight, SKColorType.Rgba8888, SKAlphaType.Premul);
+			var info = new SKImageInfo(pixelWidth, pixelHeight, SKColorType.Bgra8888, SKAlphaType.Premul);
 
 			fixed (byte* pData = data)
```

Comment:
This only locks the pixels in place for this method. Afterwards, the GC is free to move them and it will break `SKImage.FromPixels` as that just wraps the pixels without a copy. If you want a copy, use `SKImage.FromPixelCopy` instead.

Reply:
Thanks! I've adjusted this so the content is properly copied, and not too many times.